pmm-app/src/pmm-dbaas/components/XtraDB/XtraDB.service.ts

```diff
@@ -7,7 +7,17 @@ export const XtraDBService = {
     return apiRequestManagement.post<any, any>('/DBaaS/XtraDBClusters/List', kubernetes);
   },
   addXtraDBCluster(xtradbCluster: XtraDBCluster) {
-    return apiRequestManagement.post<XtraDBClusterAPI, any>('/DBaaS/XtraDBCluster/Create', toAPI(xtradbCluster));
+    return apiRequestManagement.post<XtraDBClusterAPI, any>(
+      '/DBaaS/XtraDBCluster/Create',
+      toAPI(xtradbCluster),
+    );
+  },
+  deleteXtraDBClusters(xtradbCluster: XtraDBCluster) {
+    const toAPI = (cluster) => ({
+      name: cluster.clusterName,
```

Comment:
We need to path both `name` and `kubernetes_cluster_name` because the request receives both. I think kubernetesClusterName is empty in XtraDBCluster, it may require some refactoring in the `getXtraDBClusters` or to be added in the BE as return of the list request.

Reply:
Oh, thanks for this details